examples/federated-learning-mistnet-yolo-aggregator.Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM tensorflow/tensorflow:1.15.4
+
+RUN apt update \
+  && apt install -y libgl1-mesa-glx
+
+COPY ./lib/requirements.txt /home
+
+RUN python -m pip install --upgrade pip
+
+RUN pip install -r /home/requirements.txt
+
+ENV PYTHONPATH "/home/lib:/home/plato:/home/plato/packages/yolov5"
+
```

Comment:
more advised to use `plato` as an installation package.

Reply:
> more advised to use `plato` as an installation package.

+1. `plato` as a python package instead of a submodule.

Reply:
Due to the large package issue, we currently add Plato as third-party lib.